Makefile

```diff
@@ -61,7 +61,11 @@ $(OUT)/syscall_sdl.o: CFLAGS += $(shell sdl2-config --cflags)
 LDFLAGS += $(shell sdl2-config --libs)
 endif
 
-ENABLE_GDBSTUB ?= 1
+ifeq ($(filter tool,$(MAKECMDGOALS)),tool)
```

Comment:
Do do this. Minimize the necessary changes. It does not harm if gdb-stub is built with `tool` target.